packages/edit-site/src/components/sidebar-navigation-screen-details-footer/index.js

```diff
@@ -13,14 +19,38 @@ import {
 	SidebarNavigationScreenDetailsPanelLabel,
 	SidebarNavigationScreenDetailsPanelValue,
 } from '../sidebar-navigation-screen-details-panel';
+import SidebarNavigationItem from '../sidebar-navigation-item';
 
 export default function SidebarNavigationScreenDetailsFooter( {
-	lastModifiedDateTime,
+	record,
+	...otherProps
 } ) {
+	/*
+	 * There might be other items in the future,
+	 * but for now it's just modified date,
+	 * so return null if there's no record?.modified.
+	 */
+	if ( ! record?.modified ) {
+		return null;
+	}
+
+	const hrefProps = {};
+	if ( record?._links?.[ 'predecessor-version' ]?.[ 0 ]?.id ) {
+		hrefProps.href = addQueryArgs( 'revision.php', {
+			revision: record?._links[ 'predecessor-version' ][ 0 ].id,
+			gutenberg: true,
```

Comment:
What does the `gutenberg: true` query arg do? I assume this is part of the follow-up work for revisions. What will we need to change prior to this making it in for 6.4?

Reply:
Great question. I think folks keep copying it over. I left it in there, but I wasn't sure and because there are a couple of other examples. 

I guessed it's technically there to support very old version of Gutenberg in order to discern between backlinks: https://github.com/WordPress/gutenberg/pull/3511/files#diff-e50f9b4c9183a86d62e4277f4746bb7738dd45c8f6ac50dc563ef7028681832bR493, but that code doesn't exist anymore in the plugin at least. I can't find it in Core either.

So I suppose it's safe to remove it from this PR. I can get another one up to remove it completely.

Reply:
Having said that, we might need some Core-safe queries in the future to make sure the "Go to editor" works from revisions.php, that is, that is returns to the site editor and not the edited post

Reply:
Ah, gotcha, thanks for the explanation!